src/content/developers/tutorials/how-to-view-nft-in-metamask/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: How to View Your NFT in Your Wallet (Part 3/3 of NFT Tutorial Series)
+description: This tutorial idescribes how to view an existing an NFT on your virtual Ethereum wallet (Metamask)! If you’d like to learn how to mint your own NFT on Ethereum, you should check out [Part 1 on How to Write & Deploy an NFT smart contract](/developers/tutorials/how-to-write-and-deploy-an-nft)!
```

Comment:
This description field doesn't get rendered (metadata)... I would cut this down to `This tutorial describes how to view an existing an NFT on MetaMask!`

On that note: "idescribes" typo, and MetaMask uses two capital M's